Makefile

```diff
@@ -57,35 +65,51 @@ $(LIBTRANSISTOR_HOME)/build/test/%.nso.so: $(LIBTRANSISTOR_HOME)/build/test/%.o
 $(LIBTRANSISTOR_NRO_LIB): $(LIBTRANSISTOR_HOME)/build/lib/crt0.nro.o $(libtransistor_OBJECT_FILES)
 	mkdir -p $(@D)
 	rm -f $@
-	ar rcs $@ $+
+	$(AR) $(AR_FLAGS) $@ $+
 
 $(LIBTRANSISTOR_NSO_LIB): $(LIBTRANSISTOR_HOME)/build/lib/crt0.nso.o $(libtransistor_OBJECT_FILES)
 	mkdir -p $(@D)
 	rm -f $@
-	ar rcs $@ $+
+	$(AR) $(AR_FLAGS) $@ $+
 
 $(LIBTRANSISTOR_HOME)/newlib/Makefile:
 	cd $(LIBTRANSISTOR_HOME)/newlib; ./configure --target=aarch64-none-switch --without-rdimon
 
 $(LIBTRANSISTOR_HOME)/newlib/aarch64-none-switch/newlib/libc.a: $(LIBTRANSISTOR_HOME)/newlib/Makefile
-	make -C $(LIBTRANSISTOR_HOME)/newlib/
+	$(MAKE) -C $(LIBTRANSISTOR_HOME)/newlib/
```

Comment:
Change to use `$(MAKE)` instead of just `make`. See ftp://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_5.html#SEC51